app/models/user.rb

```diff
@@ -55,6 +55,19 @@ def send_mail_activate
     UserMailer.account_activation(self).deliver_now
   end
 
+  def create_reset_digest
+    self.reset_token = User.new_token
+    update reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```

Comment:
```suggestion
    update_columns reset_digest: User.digest(reset_token), reset_sent_at: Time.zone.now
```
dùng thèn này mới hợp lý em nhé
lý do thì mạnh dạn search sự khác nhau của update column và update nhé